website/client-old/js/services/userServices.js

```diff
@@ -190,6 +190,8 @@ angular.module('habitrpg')
         })
       }
 
+      //Calling this multiple times rapidly will not persist changes
```

Comment:
What I saw was that calling it twice somehow didn't save the user object locally which was then synced. Calling the function with attribute A and then B only saves B for some reason. However if you call the function with A and B then both save.